assets/src/styles/layouts/_page.scss

```diff
@@ -40,7 +40,6 @@ body:not(.home) #content h2.denied-title {
 
 #content p.h5p-count {
   font-family: $font-family-sans-serif;
-  text-align: center;
```

Comment:
Removed center text alignment for aesthetics and so the text positioning doesn't change when the button text toggles.